repository/t.json

```diff
@@ -2653,6 +2653,18 @@
 				}
 			]
 		},
+		{
+			"name": "Tmux 3 Syntax Highlight",
+			"details": "https://github.com/Edditoria/tmux-sublime",
+			"author": "Edditoria",
```

Comment:
```suggestion
```

Since this is your GitHub username, it's better not to specify it. Package Control will pick it up automatically.